pkg/certificate/mrc_reconciler.go

```diff
@@ -153,6 +159,33 @@ func (m *Manager) updateMRCComponentStatus(mrc *v1alpha2.MeshRootCertificate) er
 	return m.mrcClient.UpdateMeshRootCertificate(mrc)
 }
 
+// checks if the given MRC meets the attributes to move the componentStatuses to Validating
+func shouldChangeStatusToValidating(mrc *v1alpha2.MeshRootCertificate) bool {
+	// case-insensitive check for if strings are equal
+	if strings.EqualFold(string(mrc.Spec.Intent), constants.MRCIntentPassive) {
+		acceptedCond := getCondition(mrc, constants.MRCConditionTypeAccepted)
```

Comment:
suggestion: the code should either declare its intent through intuitive function names, or add comments explaining how/why its performing these checks. From reading this I'm not sure what it's supposed to be doing